var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -142,7 +134,7 @@ def url_for_version(self, version):
 
     # NetCDF 4.4.0 and prior have compatibility issues with HDF5 1.10 and later
     # https://github.com/Unidata/netcdf-c/issues/250
-    depends_on('hdf5@:1.8.999', when='@:4.4.0')
+    depends_on('hdf5@:1.8.999+hl', when='@:4.4.0')
```

Comment:
Can you explain this change?

Reply:
@wardf, does this look good?

Reply:
@skosukhin @WardF would either of you like to be listed as maintainers for this package?

Reply:
Sorry, I need to run. But there is some problem with the concretizer. Just try to run:
```
spack spec netcdf-c@4.4.0
```

Reply:
I would be happy to be listed, although neither my python nor spack backgrounds are particularly strong (I am working on coming up to speed, however).

Reply:
No worries, it's just nice to have an actual developer who can take a look at build failures or recommend good defaults for the package.